docs/posts/lms-for-data/index.qmd

```diff
@@ -0,0 +1,691 @@
+---
+title: "Using language models for data"
+author: "Cody Peterson"
+date: "2024-02-05"
+image: thumbnail.png
+categories:
+    - blog
+    - llms
+    - duckdb
+---
+
+## Overview
+
+This post will give an overview of how (large) language models (LMs) fit into
+data engineering, analyst, and science workflows.
```

Comment:
```suggestion
data engineering, data analyst, and data science workflows.
```

I understand the implied `data`, but it reads a bit like "science workflows" which means something very different than "data science workflows" 😄